### DISCUSSION TITLE
[A-L] (2023/24) Foglio 3 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#152] [Esercizi - Bertini] [answered]
![Es 4](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/16648451-222f-4a16-a779-a79950f236b0)

**Comment by Elia-Belli** [comment]
Bisogna dimostrare che ogni elemento $x$ contenuto nell'unione di tutti gli insiemi $A_i$ è contato esattamente una volta.
L'elemento $x$ sarà contenuto almeno da un insieme e al massimo da tutti, posso riordinare gli $n$ insiemi in modo che i primi $j$ contengano $x$ e i successivi no. 

Inizialmente $x$ verrà contato $j= \binom{j}{1}$ volte quando sommiamo le cardinalità dei singoli insiemi;
Quando sottraggo le $\cap$ 2 a 2, gli insiemi che contengono $x$ potranno formare tra di loro $\binom{j}{2}$ coppie;
Quando aggiungo le $\cap$ 3 a 3, gli insiemi che contengono $x$ potranno formare tra di loro $\binom{j}{3}$ trii;
E così via, fino all'intersezione di tutti gli insiemi che lo contengono, che lo conta $1= \binom{j}{j}$ volte.
Quindi il numero di volte che $x$ è contato sono: $$\sum^j_{h=1}(-1)^{h-1}\binom{j}{h}$$
Voglio utilizzare il Binomio di Newton, allora per far partire la sommatoria da $0$ sottraggo il valore in più che si va a produrre, inoltre voglio trasformare $(-1)^{h-1}$ in $(-1)^h$, quindi nego tutta la sommatoria per rimettere i segni corretti: $$1-\sum^{j}_{h=0}(-1)^{h}\binom{j}{h}= 1 - (-1+1)^j=1-0=1$$
Nella formula del Binomio appare $a^{n-k}$ che nel nostro caso è $1$ (sottointeso), e $b^k$ che è il nostro $-1$.
Quindi abbiamo ottenuto che $x$ è contato esattamente $1$ volta, che è ciò che volevamo dimostrare.

**Comment by CiottoloMaggico** [comment, marked as answer]
# Soluzione per Induzione confermata in classe
**Dimostrazione per induzione su n**
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/5778164a-6bc8-4b5c-8f64-21db88e9dee6)
